src/lib/options.js

```diff
@@ -81,6 +81,7 @@ function defaultOptions() {
         version,
         secureHeaders: process.env.SECURE_HEADERS || false,
         enableOAS: process.env.ENABLE_OAS || false,
+        dbSession: process.env.DB_SESSION || true,
```

Comment:
could we introduce a session section? `{ session: { db: true }}`

Reply:
Sure, and move maxAge in there as well then?

Reply:
introduced in a9c198d4ec9dc41debd97ebe44fecb0e51b0a701